ffmpeg3/README.md

```diff
@@ -0,0 +1,15 @@
+# ffmpeg
+
+A complete, cross-platform solution to record, convert and stream audio and video.
+
+## Maintainers
+
+* The Habitat Maintainers: <humans@habitat.sh>
+
+## Type of Package
+
+Binary package
+
+## Usage
+
+*TODO: Add instructions for usage*
```

Comment:
Lets add the instructions for usage here. I think if we're adding a README file to the repo, this usage field shouldn't be left blank any more.